DoseMathTests/DoseMathTests.swift

```diff
@@ -423,7 +423,7 @@ class RecommendTempBasalTests: XCTestCase {
             lastTempBasal: nil
         )
 
-        XCTAssertEqual(1.425, dose!.unitsPerHour, accuracy: 1.0 / 40.0)
+        XCTAssertEqual(1.60, dose!.unitsPerHour, accuracy: 1.0 / 40.0)
```

Comment:
Why are any of these values changing?

Reply:
If it is because you changed the delay from the previous default to 0, consider explicitly setting the delay above to what the default was to show no changes to the algorithm (even after adding the explicit delay property to the insulin model).

Reply:
I'll change one of them back to run with a 0 delay, to show what you want to see ; the rest of them will exercise what is now what is actually used in the running app.